src/implementation/Client/GRPCClient/lock.ts

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import GRPCClient from './GRPCClient';
+import { TryLockResponse as TryLockResponseResult } from '../../../types/lock/TryLockResponse';
+import { UnlockResponse as UnLockResponseResult, LockStatus } from '../../../types/lock/UnlockResponse';
+import { TryLockRequest, TryLockResponse, UnlockRequest, UnlockResponse } from '../../../proto/dapr/proto/runtime/v1/dapr_pb';
+import IClientLock from '../../../interfaces/Client/IClientLock';
+
+export default class GRPCClientLock implements IClientLock {
+    client: GRPCClient;
+
+    constructor(client: GRPCClient) {
+        this.client = client;
+    }
+
+    async tryLock(storeName: string, resourceId: string, lockOwner: string, expiryInSeconds: number): Promise<TryLockResponseResult> {
+        const request = new TryLockRequest();
+        request.setStoreName(storeName);
+        request.setResourceId(resourceId);
+        request.setLockOwner(lockOwner);
+        request.setExpiryinseconds(expiryInSeconds);
```

Comment:
```suggestion
        const request = new TryLockRequest()
          .setStoreName(storeName)
          .setResourceId(resourceId)
          .setLockOwner(lockOwner)
          .setExpiryinseconds(expiryInSeconds);
```
This should also work.

Reply:
Same for other places.